src/components/OrgProfile.re

```diff
@@ -179,18 +179,33 @@ module OrgPage = {
       React.useState(() => None);
     let orgAnimalsArray = orgAnimals->Array.map(animal => animal##id);
     let currentUsdEthPrice = UsdPriceProvider.useUsdPrice();
-    let totalCollected = QlHooks.useTotalRaisedAnimalGroup(orgAnimalsArray);
+    let (totalCollectedMainnetEth, totalCollectMaticDai) =
+      QlHooks.useTotalRaisedAnimalGroup(orgAnimalsArray);
 
-    let (totalPatronage, totalPatronageUsd) =
-      totalCollected->mapd(("Loading", "Loading"), a =>
-        (
-          (a->Eth.get(Eth.Eth(`ether))->Float.fromString |||| 0.0)
-          ->toFixedWithPrecisionNoTrailingZeros(~digits=9),
-          currentUsdEthPrice->mapd("Loading", usdEthRate =>
-            a->Eth.get(Eth.Usd(usdEthRate, 2))
-          ),
+    let (totalPatronageUsd, totalBreakdownString) =
+      switch (totalCollectedMainnetEth, totalCollectMaticDai) {
+      | (Some(mainnetEth), Some(maticDai)) => (
+          (
+            currentUsdEthPrice->Option.mapWithDefault(0., usdEthRate =>
+              mainnetEth->Eth.getFloat(Eth.Usd(usdEthRate, 2))
+            )
+            +. maticDai->Eth.getFloat(Eth.Eth(`ether))
+          )
+          ->Js.Float.toFixedWithPrecision(~digits=6),
+          mainnetEth->Web3Utils.fromWeiBNToEthPrecision(~digits=4)
+          ++ " ETH + "
+          ++ maticDai->Web3Utils.fromWeiBNToEthPrecision(~digits=2)
+          ++ " DAI",
```

Comment:
This was another (unrelated) fix, now it shows the org profiles Dai and Eth earning.

We can definitely make this better, but seems good for now.